### ISSUE TITLE
[Question] Is there a way to reference the object in the callback?

### ISSUE THREAD

**Issue by Yaojian** [TooTallNate/node-weak#69]
I tried to execute a function which belongs to the weak referenced object inside the callback. Is it possible?

The callback passed in `weak(Object obj [, Function callback])` does not receive any argument when it is invoked, while referencing the `obj` inside the callback directly will prevent the callback being invoked as the `obj` still has active reference and cannot be GCed.

**Comment by acarstoiu** [comment]
Yes, there is, see the `README.md` file. What you're looking for is the `weak.get(ref)` function.

**Comment by mvduin** [comment]
Calling `weak.get(ref)` inside the callback results in a segfault.
